changelogs/unreleased/2949-sseago

```diff
@@ -0,0 +1 @@
+Restore lowercase plural resource name for CRDs.
```

Comment:
I might rephrase this to talk about the wait behavior; from a user standpoint the lowercase/plural doesn't mean much.